stdlib/public/core/DoubleWidth.swift.gyb

```diff
@@ -60,26 +61,31 @@ public struct DoubleWidth<Base : FixedWidthInteger> :
   internal var _storage: (low: Low, high: High)
 #endif
 
+  /// The high part of the value.
   @_inlineable // FIXME(sil-serialize-all)
   @_transparent
   public var high: High {
     return _storage.high
   }
 
+  /// The low part of the value.
   @_inlineable // FIXME(sil-serialize-all)
   @_transparent
   public var low: Low {
     return _storage.low
   }
 
+  /// Creates a new instance from the given tuple of high and low parts.
+  ///
+  /// - Parameter value: The tuple to use as the source of the new instance's
+  ///   high and low parts.
   @_inlineable // FIXME(sil-serialize-all)
   @_transparent
-  public // @testable
-  init(_ _value: (High, Low)) {
+  public init(_ value: (high: High, low: Low)) {
```

Comment:
Although it appears that this initializer was originally not intended for public consumption, the [documentation](https://github.com/apple/swift/pull/14295/files#diff-1a1ccf1c510bfc49a19be3eb3c378a8fL29) tells users to use it. In fact, this initializer is promoted as part of the main use case for `DoubleWidth` (and I would agree with that characterization).

Reply:
I didn't even notice that this wasn't a public initializer when I wrote that—seems like a pretty intuitive usage of the type. 

@moiseev, what do you think of making this public API?

Reply:
I agree we should just remove the underscore from `_value` and a `// @testable` comment. This initializer is quite useful and should be a part of `DoubleWidth` API.

Reply:
Great. I think this PR is ready then :)